src/api2/routes/authentication/getAuthenticationChallenges.ts

```diff
@@ -14,7 +14,15 @@ export const getAuthenticationChallenges2 = async (req: any, res: any) => {
     }
     const deviceId = inputSanitizer.getString(req.body?.deviceId);
     const userEmail = inputSanitizer.getLowerCaseString(req.body?.userEmail);
-    const groupIds = await getGroupIds(req);
+
+    let groupIds;
+    try {
+      groupIds = await getGroupIds(req);
+    } catch (e) {
```

Comment:
what kind of unwanted exception could we have here, that could potentielly send a "revoked_by_admin" aswell?